data/ecosystem/vendors.yaml

```diff
@@ -170,7 +170,7 @@
   commercial: true
 - name: ServiceNow Cloud Observability (Lightstep)
   nativeOTLP: true
-  url: https://github.com/lightstep?q=launcher
+  url: https://docs.lightstep.com/docs/about-sending-data
   contact:
```

Comment:
@AnaMMedina21 would you mind adding a contact here? This is for us to reach out (e.g. if the link breaks or if we need some changes made here, a github handle is preferred, an email address is also OK)

Reply:
done!